spec/system/requests/scheduling_requests_spec.rb

```diff
@@ -15,6 +15,7 @@
     it 'schedules a future job' do
       visit new_request_path(as: user)
 
+      expect(page).not_to have_link(I18n.t('components.request_form.planned_request.destroy'))
```

Comment:
Checking for some text not to present on a website is an assertion that quickly turns into a false-negative. It's too easy for some random text not to be present on a website.

My suggestion is to either delete the assertion or to have a test case where the same link is in fact asserted to be present.

Reply:
I agree that it can quickly turn into a false negative. For example, if the button text changes, the test could still pass even though the button is actually visible. I tried to mitigate this by using the `I18n` translation so that if we do change the text of the button, it would also change the test assertion.

I do have a test where the link is asserted to be present. This line, https://github.com/tactilenews/100eyes/pull/1762/files#diff-1ca2f0f20382968b26e4448455a8a781d0a7f9a02acbb1e54c39b24cc355f51dR40

```rb
click_on I18n.t('components.request_form.planned_request.destroy')
```
will fail if there is no button to click.

The reason I added the test is that we conditionally render the delete button for requests that have been created and are able to be edited. Maybe, I could write a component spec instead?

Reply:
See https://github.com/tactilenews/100eyes/pull/1762/commits/e87fb99d82d9fa6d029ac35d2ffa6a300ea2d8e6